Doc/library/tkinter.rst

```diff
@@ -188,6 +188,568 @@ the modern themed widget set and API::
+
+   .. method:: nametowidget(name)
+
+      Return the Tkinter instance of a widget identified by
+      its Tcl name NAME.
+
+   .. method:: configure(cnf=None, **kw)
+
+      Configure resources of a widget.
+
+      The values for resources are specified as keyword
+      arguments. To get an overview about
+      the allowed keyword arguments call the method keys.
+
+   .. method:: event_add(virtual, *sequences)
+
+      Bind a virtual event VIRTUAL (of the form <<Name>>)
+      to an event SEQUENCE such that the virtual event is triggered
+      whenever SEQUENCE occurs.
+
+   .. method:: event_delete(virtual, *sequences)
+
+      Unbind a virtual event VIRTUAL from SEQUENCE.
+
+   .. method:: event_generate(sequence, **kw)
+
+      Generate an event SEQUENCE. Additional
+      keyword arguments specify parameter of the event
+      (e.g. x, y, rootx, rooty).
+
+   .. method:: event_info(virtual=None)
+
+      Return a list of all virtual events or the information
+      about the SEQUENCE bound to the virtual event VIRTUAL.
+
+   Methods inheritted from :class:`WM` parent class:
+
+   .. method:: aspect(minNumer=None, minDenom=None, maxNumer=None, maxDenom=None)
+
+      Instruct the window manager to set the aspect ratio (width/height)
+      of this widget to be between MINNUMER/MINDENOM and MAXNUMER/MAXDENOM. Return a tuple
+      of the actual values if no argument is given.
+
+   .. method:: attributes(*args)
+
+      This subcommand returns or sets platform specific attributes
+
+      The first form returns a list of the platform specific flags and
+      their values. The second form returns the value for the specific
+      option. The third form sets one or more of the values. The values
+      are as follows:
+
+      On Windows, -disabled gets or sets whether the window is in a
+      disabled state. -toolwindow gets or sets the style of the window
+      to toolwindow (as defined in the MSDN). -topmost gets or sets
+      whether this is a topmost window (displays above all other
+      windows).
+
+      On Macintosh, XXXXX
+
+      On Unix, there are currently no special attribute values.
+
+   .. method:: client(name=None)
+
+      Store NAME in WM_CLIENT_MACHINE property of this widget. Return
+      current value.
+
+   .. method:: colormapwindows(*wlist)
+
+      Store list of window names (WLIST) into WM_COLORMAPWINDOWS property
+      of this widget. This list contains windows whose colormaps differ from their
+      parents. Return current list of widgets if WLIST is empty.
+
+   .. method:: command(value=None)
+
+      Store VALUE in WM_COMMAND property. It is the command
+      which shall be used to invoke the application. Return current
+      command if VALUE is None.
+
+   .. method:: deiconify()
+
+      Store VALUE in WM_COMMAND property. It is the command
+      which shall be used to invoke the application. Return current
+      command if VALUE is None.
```

Comment:
```suggestion
      Deiconify this widget. If it was never mapped it will not be mapped.
      On Windows it will raise this widget and give it the focus.
```
Pasted.